src/unitycatalog/ai/databricks.py

```diff
@@ -1,42 +1,44 @@
+import base64
 import inspect
 import json
-import base64
 import logging
+import os
 import re
 import time
 from dataclasses import dataclass
+from decimal import Decimal
 from io import StringIO
 from typing import TYPE_CHECKING, Any, Dict, List, Optional, Set
-from decimal import Decimal
 
 from typing_extensions import override
 
 from unitycatalog.ai.client import BaseFunctionClient, FunctionExecutionResult
+from unitycatalog.ai.paged_list import PagedList
 from unitycatalog.ai.utils import (
     column_type_to_python_type,
     convert_timedelta_to_interval_str,
     is_time_type,
-    validate_param,
     validate_full_function_name,
+    validate_param,
 )
-from unitycatalog.ai.paged_list import PagedList
 
 if TYPE_CHECKING:
     from databricks.sdk import WorkspaceClient
-    from databricks.sdk.service.sql import StatementState
     from databricks.sdk.service.catalog import (
         CreateFunction,
         FunctionInfo,
         FunctionParameterInfo,
     )
-    from databricks.sdk.service.sql import StatementParameterListItem
+    from databricks.sdk.service.sql import StatementParameterListItem, StatementState
 
 EXECUTE_FUNCTION_ARG_NAME = "__execution_args__"
 DEFAULT_EXECUTE_FUNCTION_ARGS = {
     "wait_timeout": "30s",
     "row_limit": 100,
     "byte_limit": 4096,
 }
+UNITYCATALOG_AI_CLIENT_EXECUTION_TIMEOUT = "UNITYCATALOG_AI_CLIENT_EXECUTION_TIMEOUT"
+DEFAULT_UC_AI_CLIENT_EXECUTION_TIMEOUT = "120"
```

Comment:
Not a blocker, but I'd recommend put these environment variables in one place like mlflow.

Reply:
I'll put this inside this file for now, just in case there're other env vars coming from OSS integration. If that's true, we'd better add a prefix if unifying them in one place? Wondering if that would be quite a long name